src/models/rpc/evm/mod.rs

```diff
@@ -5,10 +5,18 @@ use utoipa::ToSchema;
 #[serde(untagged)]
 pub enum EvmRpcResult {
     GenericRpcResult(String),
+    RawRpcResult(serde_json::Value),
 }
 
 #[derive(Debug, Serialize, Deserialize, ToSchema, PartialEq)]
-#[serde(tag = "method", content = "params")]
+#[serde(untagged)]
 pub enum EvmRpcRequest {
-    GenericRpcRequest(String),
+    GenericRpcRequest {
```

Comment:
What is the different between these 2 variants?

Reply:
In convert_to_internal_rpc_request I see we create `RawRpcRequest`, is `GenericRpcRequest` needed at all?